
Namespace(D_lr=0.0001, base_lr=0.01, batch_size=4, beta=0.3, consistency=1.0, consistency_rampup=40.0, consistency_type='kl', consistency_weight=0.1, deterministic=1, ema_decay=0.99, exp='LA/SSL_DTC_VNet_DenseModi2_concat', gamma=0.5, gpu='1', labeled_bs=2, labelnum=16, max_iterations=6000, root_path='/data/sohui/LA_dataset/2018LA_Seg_TrainingSet', seed=1337, with_cons='without_cons')
  0%|                                         | 0/751 [00:00<?, ?it/s]
total 80 samples
  0%|                                         | 0/751 [00:01<?, ?it/s]
Traceback (most recent call last):
  File "/home/sohui/.pycharm_helpers/pydev/pydevd.py", line 1483, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/sohui/.pycharm_helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/sohui/code/DTC/code/SSL_train_la_dtc_VNet.py", line 199, in <module>
    outputs_tanh, outputs = model(volume_batch)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sohui/code/DTC/code/networks/vnet_DenseModi2_concat_sdf.py", line 270, in forward
    out_tanh, out_seg = self.decoder(features)
  File "/home/sohui/code/DTC/code/networks/vnet_DenseModi2_concat_sdf.py", line 235, in decoder
    x6 = self.block_six(x5_up)  # 4,128,14,14,10   #conv -> BN -> Relu (ch 1/2)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sohui/code/DTC/code/networks/vnet_DenseModi2_concat_sdf.py", line 27, in forward
    x = self.conv(x)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/torch/nn/modules/conv.py", line 572, in forward
    return F.conv3d(input, self.weight, self.bias, self.stride,
RuntimeError: Given groups=1, weight of size [128, 256, 3, 3, 3], expected input[4, 128, 14, 14, 10] to have 256 channels, but got 128 channels instead